jupyter_releaser/lib.py

```diff
@@ -54,6 +54,9 @@ def check_links(ignore_glob, ignore_links, cache_file, links_expire):
     cmd += f"--check-links-cache-name {cache_dir}/check-release-links "
     # do not run doctests, since they might depend on other state.
     cmd += "-p no:doctest "
+    # ignore package pytest configuration,
+    # since we aren't running their tests
+    cmd += "-c _IGNORE_CONFIG"
```

Comment:
This isn't a special value for pytest; pytest has no option I could find to skip checking config file names, so this is a filename that probably won't exist, so no config will be found.

It could also be done to put the check-links pytest config in a config file in this package, and use that to override the package's config.

Reply:
Yeah I think since the config is dynamic we should write out a temporary config file, echo it for debugging, and then use it during each run.  This is a reasonable workaround for now, thanks!